en/style/material-icons.md

```diff
@@ -74,6 +80,36 @@ With the preset collection of icons, you can design beautiful apps and pick the
 
     <img class="responsive-img" src="../images/icons_add5.png" srcset="../images/icons_add5@2x.png 2x" />
 
+### In Adobe XD
+
+1.  Open up the library in Adobe XD navigate to the `🎨 Material Icons` artboard and look if that same category exists as a text layer on the left. If it does not, create it and follow the steps below. If it does, which is our case for the `copyright` icon from the `Action` category, just find the right-most icon from that category and you should be looking at something like this.
+
+    <img class="responsive-img" src="../images/icons_add6.png" srcset="../images/icons_add6@2x.png 2x" />
+
+2.  Now, duplicate the Symbol for this right-most icon and move it right a bit. This will create an instance of the symbol indicated by the white diamond shape with green outline in the top-left corner when the instance is selected.
+
+    <img class="responsive-img" src="../images/icons_add7.png" srcset="../images/icons_add7@2x.png 2x" />
+
+3.  Ungroup this instance and create a new group from the two new layers, naming it "_Overrides/Material Icons/action/copyright".
+    | Layer | Use |
+    | ----------------------- | ----------------------------------------------------- |
+    | Group ~ | Defines the icon shape and color |
+    | Shape | Always a square that defines the icon bounding box |
+    <div class="divider--half"></div>
+    <img class="responsive-img" src="../images/icons_add8.png" srcset="../images/icons_add8@2x.png 2x" />
+
+4.  Next, we want to replace the existing icon shape with the SVG we have downloaded, so we need to select the Group with the old glyph and drag and drop the SVG with the `copyright` icon on top of it aligning the two on all sides. This will create a new group `copyright-24px` in our layers panel, which we need to expand and drag the desired icon shape just outside of this group, but above the rectangular Shape layer.
```

Comment:
```suggestion
4.  Next, we want to replace the existing icon shape with the SVG we have downloaded, so we need to select the Group with the old glyph and drag and drop the SVG with the `copyright` icon on top of it, aligning the two on all sides. This will create a new group, `copyright-24px` in our layers panel, which we need to expand and drag the desired icon shape just outside of this group, but above the rectangular Shape layer.
```